pico_menu_rotary_test.py

```diff
@@ -33,6 +33,14 @@
 
 # fonts = [glcdfont,tt14,tt24,tt32]
 
+# Colors
+RED = (255, 0, 0)
+GREEN = (0, 255, 0)
+BLACK = (0, 0, 0)
+
+# Pos
+POS = [(45, 20), (45, 60), (45, 100), (45, 140)]
```

Comment:
There are four positions each with the same x value (45) but different y values.